cwl_utils/parser/cwl_v1_2_utils.py

```diff
@@ -10,6 +10,24 @@
 from cwl_utils.errors import WorkflowException
 
 
+CONTENT_LIMIT: int = 64 * 1024
+
+
+def content_limit_respected_read_bytes(f: IO[bytes]) -> bytes:
+    """Read file content up to 64 kB as a byte array."""
+    contents = f.read(CONTENT_LIMIT + 1)
+    if len(contents) > CONTENT_LIMIT:
+        raise WorkflowException(
+            "file is too large, loadContents limited to %d bytes" % CONTENT_LIMIT
+        )
+    return contents
```

Comment:
Why does only CWL v1.2 have the exception?

Reply:
Because spec says that 1.0 and 1.1 truncate content, while 1.2 raises error.

Reply:
If you prefer, we can move the constant in the  __init__.py